edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/IServiceIdentityHierarchy.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Core
 
     public interface IServiceIdentityHierarchy
     {
+        string GetActorDeviceId();
```

Comment:
I think this is not a method related to this interface.  I think we should pass edgeDeviceId to DeviceScopeIdentitiesCache directly instead.

Reply:
The hierarchy is the source of truth for caching and getting all things identity related, i.e. all IDs, properties, and relationships between them.  The actor is just a special identity that's the root of the hierarchy.

Reply:
Say we have childEdge, parentEdge and rootEdge.  When requests come from childEdge to parentEdge.  The authChain returns for childEdge should be "childEdge;parentEdge;rootEdge", not "childEdge;parentEdge", do i understand correctly?